### DISCUSSION TITLE
Setting `clientMeta` in WalletConnect

### DISCUSSION THREAD

**Discussion by czxtm** [rainbow-me/rainbowkit#816] [Q&A] [answered]
WalletConnect lets you pass `clientMeta` to customize the title, description, and icon that is shown in the "Do you want to connect to..." screen in most wallets. How do we set this in RainbowKit?

**Comment by banky** [comment]
Hey @coopermaruyama, did you find a way to do this? I'm also having the same issue. For me, the icon just shows up as the characters of my app name

**Reply by czxtm** [reply, marked as answer]
Yeah I worked around it but the solution is not ideal whatsoever -- WalletConnect by default will get this metadata from the meta tags in the `<head>` tag of your website. For example, the favicon or `<link rel="icon">` , or `<meta name="description">` for the description. Somewhere buried in the walletconnect repos is the exact details of which tags it checks and in which order. So I just modified those tags to change the icon.
